README.rst

```diff
@@ -53,7 +53,7 @@ To install ARIA from source, download the source tarball from
 **Ubuntu/Debian** (tested on Ubuntu14.04, Ubuntu16.04)::
 
     apt-get install -y python-dev gcc libffi-dev libssl-dev
-    pip install aria[ssh]
+    pip install apache-ariatosca[ssh]
```

Comment:
Thanks. Now I just have to ask that you squash the two commits into a single one, and make sure you're rebased on top of master once you do that - then I'll be able to merge this.